code/modules/power/cells/power_cells_vr.dm

```diff
@@ -0,0 +1,7 @@
+/obj/item/weapon/cell/recharging
+	name = "recharging high-capacity power cell"
+	origin_tech =  null
+	icon_state = "icell"
```

Comment:
cancer because it doesn't have an iconstate, nobody mouses over every single cell to see the name. there was a time when mechs and things used their own special mystical cells and had the same icons as normal tier cells and it was super annoying because people would ahelp about it.

i get that nobody is ever going to see these but still. just put some like. dots on the icon or make it different somehow.